tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
@rpelisse this hardcodes it to a linux machine. I would add a condition to ensure you are running on a linux machine